ssds/storage.py

```diff
@@ -128,6 +114,41 @@ def verify_checksums(src_url: str,
             else:
                 raise SSDSMissingChecksum(msg)
 
+def copy_oneshot_passthrough(src_blob: AnyBlob,
+                             dst_blob: CloudBlob,
+                             compute_checksums: bool=False) -> Dict[str, str]:
+    """
+    Copy from `src_blob` to `dst_blob`, passing data through the executing instance.
+    Optionally compute checksums.
+    """
+    data = src_blob.get()
+    if compute_checksums:
+        checksums = {SSDSObjectTag.SSDS_MD5: checksum.md5(data).hexdigest(),
+                     SSDSObjectTag.SSDS_CRC32C: checksum.crc32c(data).google_storage_crc32c()}
+    else:
+        checksums = dict()
+    dst_blob.put(data)
+    return checksums
+
+def copy_multipart_passthrough(src_blob: AnyBlob,
+                               dst_blob: CloudBlob,
+                               compute_checksums: bool=False) -> Dict[str, str]:
+    """
+    Copy from `src_blob` to `dst_blob`, passing data through the executing instance.
+    Optionally compute checksums.
+    """
+    if compute_checksums:
+        checksums = {SSDSObjectTag.SSDS_MD5: checksum.S3EtagUnordered(),
+                     SSDSObjectTag.SSDS_CRC32C: checksum.GScrc32cUnordered()}
+    else:
+        checksums = dict()
```

Comment:
Maybe consider an optional tag like "uncomputed" to differentiate files that have been purposely uploaded with this tool without checksums?

Reply:
This can be reconsidered in the future if needed. For now, there is an `ignore_missing_checksums` parameter that can be used to manipulate objects with missing checksums. if `ignore_missing_checksums` is set to True, SSDS will spit warnings for objects with missing checksums.